user-console/src/main/java/org/pentaho/mantle/client/admin/ChangePasswordByUserDialog.java

```diff
@@ -48,14 +48,15 @@ public class ChangePasswordByUserDialog extends GwtDialog implements ServiceCall
   private Button cancelBtn = new Button( Messages.getString( "cancel" ) );
   private static final String TEXT_BOX_WIDTH = "260px";
   private static final String SPACER_STYLE_NAME = "spacer";
+  private boolean enableAcceptBtn = false;
```

Comment:
Please rename to something like `acceptBtnEnabled`. This name should describe a _state_, and not an _action_. Also, as per your proposal, the action and the state have the same name, something I think is confusing, even though legal (for a field and a method to have the same name).